website/blog/2025-12-05-acr-abac-repository-permissions/index.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Azure Container Registry Repository Permissions with Attribute-based Access Control (ABAC)"
+date: "2025-12-05"
+description: Azure Container Registry now supports Entra ABAC conditions in RBAC role assignments. This enables identities from CI/CD pipelines and AKS clusters to have granular image push, pull, or delete permissions for specific repositories and namespaces.
+authors:
+    - johnson-shi
+tags:
+    - azure-container-registry
+    - entra
+    - best-practices
+    - general
+    - operations
+    - security
+    - workload-identity
+image: ./hero.png
+hide_table_of_contents: false
+draft: false
+---
+
+Enterprises are converging on centralized container registries that serve multiple business units and application domains. In this model, different teams share a single registry, but traditional Azure role-based access control (RBAC) forces an all-or-nothing choice: either grant broad registry-wide permissions or manage separate registries per team. Neither approach aligns with least-privilege principles or modern zero trust architectures.
```

Comment:
> but traditional Azure role-based access control (RBAC) forces an all-or-nothing choice: either grant broad registry-wide permissions or manage separate registries per team.

before this line I suggest giving a short 2-3 line intro to Azure RBAC, dataActions, scope to set the context on why before ABAC permissions was restricted to scope (and what scopes ACR offered till registry level granularity)